src/api_server.h

```diff
@@ -71,8 +74,11 @@ class MetadataApiServer {
   // Handler functions.
   void HandleMonitoredResource(const HttpServer::request& request,
                                std::shared_ptr<HttpServer::connection> conn);
+  void HandleHealthz(const HttpServer::request& request,
+                     std::shared_ptr<HttpServer::connection> conn);
 
   const Configuration& config_;
+  HealthChecker* health_checker_;
```

Comment:
Will you be modifying it? Why can't it be a const reference (or a const pointer if you accept `nullptr` as a valid value)?

Reply:
Done.